docs/runtimes/create-and-use-functions.ipynb

```diff
@@ -0,0 +1,550 @@
+    "``` {admonition} Note\n",
+    "MLRun is not limited to Python. Bash, Go, etc. files are also supported.\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "41f62728",
+   "metadata": {},
+   "source": [
+    "#### Python"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bf09c489",
+   "metadata": {},
+   "source": [
+    "This is the simplest way to create a function out of a given piece of code - simply pass in the path to the Python file *relative to your project context directory*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "af00fedd",
+   "metadata": {},
+   "source": [
+    "```python\n",
+    "fn = project.set_function(\n",
+    "    name=\"python\", func=\"job.py\",  kind=\"job\",\n",
+    "    image=\"mlrun/mlrun\", handler=\"handler\"\n",
+    ")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5ab01939",
+   "metadata": {},
+   "source": [
+    "#### Jupyter Notebook"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d5c252fd",
+   "metadata": {},
+   "source": [
+    "This is a great way to create a function out of a Jupyter notebook  - simply pass in the path to the Jupyter notebook  *relative to your project context directory*. You can use [**MLRun cell tags**](./mlrun_code_annotations.html) to specify which parts of the notebook should be included in the function. \n",
+    "\n",
+    "``` {admonition} Note\n",
+    "To ensure that the latest changes are included, make sure you save your notebook before creating/updating the function\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2c934acd",
+   "metadata": {},
+   "source": [
+    "```python\n",
+    "fn = project.set_function(\n",
+    "    name=\"notebook\", func=\"nb.ipynb\", kind=\"serving\", \n",
+    "    image=\"mlrun/ml-models\", requirements=[\"pandas==1.3.5\"]\n",
+    ")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6b36faf7",
+   "metadata": {},
+   "source": [
+    "You can also create an MLRun function out of the current Jupyter notebook you are running in. To do this, simply ommit the `func` parameter in `set_function`. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0204fd5d",
+   "metadata": {},
+   "source": [
+    "<a id=\"multiple-source-files\"></a>\n",
+    "### Multiple Source Files"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4f32f1ea",
+   "metadata": {},
+   "source": [
+    "If your code requires additional files or external libraries, you will need to use a source that supports multiple files such as Git, an archive (zip/tar/etc.), or V3IO file share. This approach (especially using a Git repo) pairs well with MLRun Projects.\n",
```

Comment:
"If your code requires additional files or external libraries, you need to use a source that supports multiple files such as Git, an archive (zip/tar/etc.), or V3IO file share. This approach (especially using a Git repo) pairs well with MLRun Projects.\n",